### PR TITLE
Drafts of posts

### PR DESCRIPTION
**Pull request by PetrushynskyiOleksii**
Ref #23 (1).

- Add page with drafts of posts
- Add delete, publish button
- Also create tests

### CODE REVIEW

File: blog/locale/uk/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -8,7 +8,7 @@ msgid ""
 msgstr ""
 "Project-Id-Version: PACKAGE VERSION\n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2018-02-10 15:49+0200\n"
+"POT-Creation-Date: 2018-02-20 11:29+0200\n"
```
Comment (webknjaz) [inline blog/locale/uk/LC_MESSAGES/django.po]: You might want to add messages generation to CI and then check whether there's new strings, for which you forgot to run and commit re-regerated po file.

File: blog/templates/blog/post_detail.html [hunk 2]
```diff
@@ -1,16 +1,23 @@
...
+{% load i18n %}
+
 {% block content %}
   <div class="post">
     {% if post.published_date %}
       <div class="date">
         {{ post.published_date }}
       </div>
+    {% else %}
+      <a class="btn btn-default" href="{% url 'post_publish' pk=post.pk %}">{% trans "Publish" %}</a>
```
Comment (webknjaz) [inline blog/templates/blog/post_detail.html:12]: Will unauthenticated dude see this?

File: blog/tests/test_views.py [hunk 3]
```diff
@@ -107,3 +107,28 @@ def test_post_edit(self):
                                     follow=True)
         self.assertContains(response, 'Test Text Edit')
         self.assertEqual(200, response.status_code)
+
+    def test_drafts(self):
```
Comment (webknjaz) [inline blog/tests/test_views.py:111]: Please learn the existence of *negative tests*

File: blog/templates/blog/post_detail.html (hunk 2 above)
Comment (PetrushynskyiOleksii) [reply blog/templates/blog/post_detail.html:12]: Unauthenticated dude cant goes to page with drafts of posts , then he doesn't see unpublished posts.
He can only manually writes links and goes to page with unpublished post (or page with drafts), but i'll fix this later

File: blog/tests/test_views.py (hunk 3 above)
Comment (PetrushynskyiOleksii) [reply blog/tests/test_views.py:111]: I added a few negative tests.
When I integrate the decorator then I will add the tests for the authentication

File: blog/templates/blog/post_detail.html (hunk 2 above)
Comment (webknjaz) [reply blog/templates/blog/post_detail.html:12]: I mean, is this link shown in their interface?

**Review by webknjaz — APPROVED** [review]

**Pull request merged**